pkg/reconciler/parser.go

```diff
@@ -156,7 +156,7 @@ func (p *Parser) parseRoleBinding(
 
 		selector, err := metav1.LabelSelectorAsSelector(&rb.NamespaceSelector)
 		if err != nil {
-			logrus.Debug("Error parsing label selector")
+			logrus.Infof("Error parsing label selector: %s", err.Error())
```

Comment:
Just calling this out because there is a chance that someone could submit an erroneous configuration for `matchExpressions` that I don't think validation in the CRD is possible to catch. Because of that this log line should always show so the user is aware.